reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -405,7 +425,35 @@ def generate_xy_plane(n_pix):
         return xy_plane
 
     @staticmethod
-    def generate_slices(map_3d_f, xy_plane, rots):
+    def generate_xyz_voxels(n_pix):
```

Comment:
I would rename this function to clarify that it gives coordinates. Perhaps `generate_xyz_cube` to match `generate_xy_plane`. 

Perhaps better to name them both generate_coords and have d=2 and d=3 as options, and refactor them into one function. They share a similar logic